libcxxabi/src/CMakeLists.txt

```diff
@@ -75,6 +75,10 @@ if (NOT APPLE) # On Apple platforms, we always use -nostdlib++ so we don't need
   add_library_flags_if(LIBCXXABI_HAS_C_LIB c)
 endif()
 
+if (FUCHSIA)
+    add_link_flags("-Wl,--push-state,--as-needed,-lzircon,--pop-state")
```

Comment:
I'd leave a `TODO` comment to use https://cmake.org/cmake/help/latest/variable/CMAKE_LINK_LIBRARY_USING_FEATURE.html once LLVM requires CMake at least 3.24.

Reply:
done.

Reply:
`TODO: $upper_case` is more common than `TODO `

Reply:
Good catch. Thx.